### PR TITLE
[ovirt-engine-rename] Fix for unresolvable hostname

### PR DESCRIPTION
**Pull request by StLuke**

### CODE REVIEW

File: tests/test-master.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@
   vars:
     ovirt_engine_version: "4.2"
     ovirt_rpm_repo: "http://plain.resources.ovirt.org/pub/yum-repo/ovirt-release-master.rpm"
-# - include: engine-rename.yml
+- include: engine-rename.yml
```
Comment (lukas-bednar) [inline tests/test-master.yml]: Please enable this test in other test-matrix 4.1, upgrade-X->Y

File: roles/ovirt-engine-rename/tasks/main.yml [hunk 2]
```diff
@@ -62,12 +62,3 @@
   command: echo Engine Rename successful
   changed_when: False
   when: final_check.rc == 0
-
```
Comment (lukas-bednar) [inline roles/ovirt-engine-rename/tasks/main.yml]: keep this check there, why did you removed it ?

File: roles/ovirt-engine-rename/tasks/main.yml [hunk 3]
```diff
@@ -51,7 +51,21 @@
...
 
-- debug: msg="change your /etc/hosts file to reflect to the new changed ovirt-engine name"
+- name: "change your /etc/hosts file to reflect to the new changed ovirt-engine name"
+  lineinfile:
+    dest: /etc/hosts
+    line: "{{ hostvars[inventory_hostname]['ansible_default_ipv4']['address'] }} {{ ovirt_engine_rename_new_fqdn}}"
+    state: present
+    unsafe_writes: "{{ovirt_run_containerized}}"
+
+- name: check health status of page
```
Comment (StLuke) [inline roles/ovirt-engine-rename/tasks/main.yml]: move to line 78

File: roles/ovirt-engine-rename/tasks/main.yml [hunk 4]
```diff
@@ -51,7 +51,21 @@
   tags:
     - skip_ansible_lint
 
-- debug: msg="change your /etc/hosts file to reflect to the new changed ovirt-engine name"
+- name: "change your /etc/hosts file to reflect to the new changed ovirt-engine name"
```
Comment (StLuke) [inline roles/ovirt-engine-rename/tasks/main.yml]: this should not be included inside role, change of /etc/hosts inside role is not desired, please move it before the role is called

File: roles/ovirt-engine-rename/defaults/main.yml [hunk 5]
```diff
@@ -0,0 +1,2 @@
+# This variable allows run rename role under container
+ovirt_run_containerized: false
```
Comment (StLuke) [inline roles/ovirt-engine-rename/defaults/main.yml]: not realted to role

File: roles/ovirt-engine-rename/tasks/main.yml [hunk 6]
```diff
@@ -70,4 +68,4 @@
   register: health_page
   retries: 12
   delay: 10
-  until: health_page|success
+  until: health_page|success
```
Comment (StLuke) [inline roles/ovirt-engine-rename/tasks/main.yml]: empty line in the end

**Comment by StLuke** [comment]
+1

**Comment by lukas-bednar** [comment]
+1

I would be rather to call rename role only once, but if this is the way you want to go, then fell free ...

**Pull request merged**